static/js/i18n/strings/en/stats_var_labels.json

```diff
@@ -87,6 +87,34 @@
     "defaultMessage": "Violent Crimes",
     "description": "Label used to describe statistical variables in chart legends, string should be short. Describes the total number of violent crimes."
   },
+  "Count_CriminalIncidents_IsHateCrime": {
+    "defaultMessage": "Hate Crime Incidents",
+    "description": "Label used to describe statistical variables in chart legends, string should be short. Describes the total number of hate crime incidents."
+  },
+  "Count_CriminalIncidents_BiasMotivationGender_IsHateCrime": {
+    "defaultMessage": "Hate Crime Incidents",
```

Comment:
A more specific label would be good - "Hate Crimes (Motivated by Gender)"

The following labels can follow the same template

Reply:
fixed.